src/Core/Shipping/BulkSenderOptions.cs

```diff
@@ -12,5 +12,8 @@ public class BulkSenderOptions
         public bool Debug { get; set; }
         public bool UseGzip { get; set; } = false;
         public string ProxyAddress { get; set; } = String.Empty;
+        
+        public string Format { get; set; } = "text";
```

Comment:
Think this should be removed from here, since the option is not about shipping, but about how to create the payload.

Reply:
It's part of the shipper's parameters, that's the common structure of our shippers

Reply:
Maybe change to a boolean called `EnableJsonMessage` (Skip constant `ToLower` and string-comparisons)